tom_mmt/mmt.py

```diff
@@ -42,6 +42,8 @@ def is_valid(self):
             self.add_error(None, errors)
         return super().is_valid()
 
+class MMTCamObservationForm(MMTBaseObservationForm):
+    program = forms.ChoiceField(choices=settings.FACILITIES['MMT']['programs']['MMTCam'])
```

Comment:
In this case, because MMTCam has only imaging, I would just combine this with the block of code below. So:
```
class MMTCamObservationForm(MMTBaseObservationForm):
    program = forms.ChoiceField(choices=settings.FACILITIES['MMT']['programs']['MMTCam'])
    filter = forms.ChoiceField(choices=[('u','u'),('g', 'g'), ('r', 'r'), ('i', 'i'), ('z', 'z')])
    exposure_time = forms.IntegerField(min_value=1, initial=100)
    number_of_exposures = forms.IntegerField(initial=5, min_value=1)
    def layout(self):
    ...
```